website/docs/reference/global-configs/legacy-behaviors.md

```diff
@@ -12,16 +12,77 @@ Another category of flags provides existing projects with a migration window for
+3. **Removal (generally enabled):** After marking the flag for deprecation, we remove it along with the 'old' behavior it supported from the dbt codebases. We aim to support most flags indefinitely, but we're not committed to supporting them forever. If we choose to remove a flag, we'll offer significant advance notice.
 
-These flags can _only_ be set in `dbt_project.yml`. They configure behaviors closely tied to project code. They should be defined in version control, and changed via pull/merge request, with the same testing and peer review.
+## Behavior change flags
 
-## Behaviors
+These flags _must_ be set in the `flags` dictionary in `dbt_project.yml`. They configure behaviors closely tied to project code, which means they should be defined in version control and modified through pull or merge requests, with the same testing and peer review.
+
+The following example displays the current flags and their current default values in the latest dbt Cloud and dbt Core versions. To opt out of a specific behavior change, set the values of the flag to `False` in `dbt_project.yml`. You'll continue to see warnings for legacy behaviors that you have opted out of explicitly until you either resolve them (switch the flag to `True`) or choose to silence the warnings.
+
+<File name='dbt_project.yml'>
+
+```yml
+flags:
+  require_explicit_package_overrides_for_builtin_materializations: False
+  require_model_names_without_spaces: False
+  source_freshness_run_project_hooks: False
+```
+
+</File>
+
+| Flag                                                            | dbt Cloud: Intro | dbt Cloud: Maturity | dbt Core: Intro | dbt Core: Maturity | 
+|-----------------------------------------------------------------|------------------|---------------------|-----------------|--------------------|
+| require_explicit_package_overrides_for_builtin_materializations | 2024.04.141      | 2024.05.xxx         | 1.6.14, 1.7.14    | 1.8.0             |
+| require_resource_names_without_spaces                           | 2024.05.xxx      | 2024.06.xxx         | 1.8.0           | 1.9.0             |
+| source_freshness_run_project_hooks                              | 2024.03.61       | 2024.06.xxx         | 1.8.0           | 1.9.0             |
+
+<sup>1</sup>dbt Cloud - "Keep on latest version"
+
+### `require_explicit_package_overrides_for_builtin_materializations`
+
+We have deprecated the behavior whereby installed packages could override built-in materializations without explicit opt-in from the end user. When this flag is set to `True`, a materialization defined in a package that matches the name of a built-in materialization will no longer be included in the search and resolution order. Unlike for macros, materializations do not use the `search_order` defined in the project `dispatch` config.
+
+The built-in materializations are `'view'`, `'table'`, `'incremental'`, `'materialized_view'` for models as well as `'test'`, `'unit'`, `'snapshot'`, `'seed'`, and `'clone'`.
```

Comment:
this makes it seem like test, unit, snapshots, are also materializations?  do we mean to say 'the builtin materializations for models are xyz. The builtin dbt resources (?)  are tests, seed, etc.

```suggestion
The built-in [materializations](/docs/build/materializations) are `'view'`, `'table'`, `'incremental'`, `'materialized_view'` for models as well as `'test'`, `'unit'`, `'snapshot'`, `'seed'`, and `'clone'`.
```

Reply:
@mirnawong1 Closing this as resolved, but let me know if anythign needs to happen here!

Reply:
oh that's interesting! thanks for sharing @jtcohen6 ! and all good, ty @runleonarun !